### DISCUSSION TITLE
Ability to rearrange Libraries Order

### DISCUSSION THREAD

**Discussion by faizanraso** [spacedriveapp/spacedrive#1555] [Feature Request]
Having the ability to rearrange libraries order is extremely important to me and is one of the first things I tried doing when getting this app.

**Comment by utkubakir** [comment]
Libraries or locations? We plan to have ordering for tags and locations on the sidebar but we could also squeeze in libraries too.

**Reply by faizanraso** [reply]
For me, it would be useful for libraries. I usually arrange my pinned folders/favourites in finder in a order that I like. So being able to do that for libraries would be great.
